internals/overlord/state/identities.go

```diff
@@ -47,28 +52,63 @@ type LocalIdentity struct {
 	UserID uint32
 }
 
+// BasicIdentity holds identity configuration specific to the "basic" type
+// (for HTTP basic authentication).
+type BasicIdentity struct {
+	Password string // Note: this is the sha512-crypt hashed password.
```

Comment:
I don't recall, will a comment after the field like this appear in docs? I suspect no, in which case I think it's probably worth putting this before and refactoring into the usual style. Would something like this work?
```
// Password holds the user's sha512-crypt-hashed password.
```

Reply:
Yes, a comment like this will be put in the docs as verified by Ben. However I do think your suggested line is better (and it fits the one on the client better as well), so I will adopt this. Thanks!